examples/Cargo.toml

```diff
@@ -24,4 +24,4 @@ webpki-roots = "=0.26.0-alpha.1"
 
 [dev-dependencies]
 regex = "1.9"
-ring = "0.16.20"
+ring = "0.17"
```

Comment:
AFAICT this and regex actually can be deleted, as neither are used in this crate

Reply:
Ah, yes, did that.